Tests/PSGetFindModule.Tests.ps1

```diff
@@ -24,12 +24,13 @@ function SuiteSetup {
     $script:MyDocumentsModulesPath = Get-CurrentUserModulesPath
     $script:PSGetLocalAppDataPath = Get-PSGetLocalAppDataPath
     $script:DscTestModule = "DscTestModule"
+    $script:PrereleaseTestModule = "TestPackage"
```

Comment:
Sure.  When we move to testing in INT, I'll create a PrereleaseTestModule.  This was done just for testing locally.